### PR TITLE
docs(managing_deis): add OpenStack Swift store example

### PR DESCRIPTION
**Pull request by duanhongyi**

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

Your Pull Request can earn you credit at DigitalOcean! See http://docs.deis.io/en/latest/contributing/hacking/#digitalocean-credit for details.

File: docs/managing_deis/running-deis-without-ceph.rst [hunk 1]
```diff
@@ -116,7 +116,9 @@ than the :ref:`logger` component.
 Configure registry
 ^^^^^^^^^^^^^^^^^^
 
-The :ref:`registry` component won't start until it's configured with an S3 store.
+The :ref:`registry` component won't start until it's configured with an store.
```
Comment (mboersma) [inline docs/managing_deis/running-deis-without-ceph.rst]: Could you change `an` to `a` here?
```The :ref:`registry` component won't start until it's configured with a store.```

File: docs/managing_deis/running-deis-without-ceph.rst [hunk 2]
```diff
@@ -137,6 +139,19 @@ The :ref:`registry` component won't start until it's configured with an S3 store
                                gateway/host=s3.amazonaws.com \
                                gateway/port=80
 
+Openstack-swift must be installed swift3 (see `Swift3`_), configuration sample:
```
Comment (mboersma) [inline docs/managing_deis/running-deis-without-ceph.rst]: Could you change this sentence to something like this?
```
Openstack-swift support requires `Swift3`_ middleware to be installed. Here is a sample configuration:
```

**Comment by duanhongyi** [comment]
@mboersma 

Review Completed !
Thank you very much, my english is very poor.

**Comment by mboersma** [comment]
@duanhongyi, excellent, thank you! Could you use `git` to squash these two commits together and change the commit message to:
```
docs(managing_deis): add OpenStack Swift store example
```

I'm sorry to focus on these little details, but we try to keep all the commits consistent according to the [Deis style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). Thanks again.

**Comment by duanhongyi** [comment]
@mboersma 

I have modified git's comments.

**Comment by mboersma** [comment]
There is an extra commit in the PR now. And your two commits should be squashed together. So you should do:
```console
$ git checkout master && git pull
$ git checkout patch-2 && git rebase master
$ git rebase -i HEAD~2
```
But the last command is not simple. It will open an editor two times, and it requires some advanced `git` knowledge. If you can use `git rebase -i` to "squash" the last two commits into one, that would be great. If not, I would be glad to do that for you and submit a new PR.

Thanks again @duanhongyi!

**Comment by duanhongyi** [comment]
@mboersma 

completed!!!

**Comment by mboersma** [comment]
Jenkins, add to whitelist.

**Comment by mboersma** [comment]
Congratulations @duanhongyi! You've earned $75 of [DigitalOcean](https://www.digitalocean.com/) credit, enough to run a [Deis](https://github.com/deis/deis) cluster for over a month.

Please email digitalocean@deis.com to confirm, and we will reply with your DigitalOcean voucher codes.

Thanks again for contributing to Deis!

**Pull request merged**